Lib/test/test_argparse.py

```diff
@@ -2018,6 +2018,15 @@ def test_parse_known_args(self):
             self.parser.parse_known_args('0.5 -W 1 b -X Y -w 7 Z'.split()),
             (NS(foo=False, bar=0.5, w=7, x='b'), ['-W', '-X', 'Y', 'Z']),
         )
+        self.assertEqual(
+            self.parser.parse_known_args(['0.5', '--opt="with space"']),
+            (NS(foo=False, bar=0.5), ['--opt="with space"']),
+        )
+        self.assertRaisesRegexp(
```

Comment:
Please do not use `assertRaisesRegexp()` -- it is deprecated.  Please use `assertRaisesRegex()`.